node-tests/e2e/test-support-test.js

```diff
@@ -185,4 +185,28 @@ describe('e2e: provides test support', function() {
 
     expect(code).to.equal(0);
   });
+
+  it('does not break development server for builds not including tests', async function() {
```

Comment:
```suggestion
 // One common scenario is when running the server in production mode locally, i.e. 
 // you are doing the production build on your local machine, connecting to your actual production server,
 // for example via `ember serve -prod`. In these cases we don't want this addon to break.
  it('does not break development server for builds not including tests', async function() {
```

Reply:
Applied that suggestion on 952f3fccb4b9bf4b2d8a80cfd6b9991a25435eec as well.